sql-cli/sql_cli/__main__.py

```diff
@@ -125,6 +125,7 @@ def run(
     project_dir: Path = typer.Option(
         None, dir_okay=True, metavar="PATH", help="(Optional) Default: current directory.", show_default=False
     ),
+    gen_dag: bool = typer.Option(default=False, help="whether to generate a DAG file", show_default=True),
```

Comment:
I find the name `gen_dag` misleading. I suggest we name the `Option` `--export-dag`. I believe, by default, we should generate the DAG - so we have a consistent way of representing what we are running locally and remotely.